src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py

```diff
@@ -597,3 +603,22 @@ def _log_webserver_credentials(self) -> None:
             "with username: `admin` password: `%s`",
             password,
         )
+
+    def get_pipeline_run_metadata(
+        self, run_id: UUID
+    ) -> Dict[str, "MetadataType"]:
+        """Get general component-specific metadata for a pipeline run.
+
+        Args:
+            run_id: The ID of the pipeline run.
+
+        Returns:
+            A dictionary of metadata.
+        """
+        run_metadata = {}
+        if self._dag_path:
+            run_metadata["airflow_dag_path"] = Path(self._dag_path)
+        if self.config.local:
+            run_metadata["airflow_log_file"] = Path(self.log_file)
```

Comment:
This is actually the server log file, I don't think that's particularly relevant for the run?

Reply:
Yeah makes sense. I didn't know which attributes make sense to log for Airflow so I just took everything that looked reasonable at first glance 😅